legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -826,6 +835,12 @@ def create_invoice(business: Business,  # pylint: disable=too-many-locals,too-ma
                 mailing_address = business.mailing_address.one_or_none()
             corp_type = business.legal_type if business.legal_type else \
                 filing.json['filing']['business'].get('legalType')
+        # deal with withdrawing a new business filing
+        elif business.identifier.startswith('T') and \
+                filing.filing_type == Filing.FILINGS['noticeOfWithdrawal']['name']:
+            mailing_address, corp_type, legal_name = \
+                ListFilingResource._get_address_from_withdrawn_new_business_filing(business, filing)
+            business.legal_name = legal_name
```

Comment:
for temporary business notice of withdraw, get mailng_address, corp_type, legal_name from the withdrawn new business filing (e.g. IA, Continuation In, Amalgamation)